lib/src/components/NxTextInput/__tests__/NxTextInput.test.tsx

```diff
@@ -61,7 +55,13 @@ describe('NxTextInput', function() {
   });
 
   it('sets the nx-tooltip--validation-error class on the tooltip', function() {
-    expect(getShallowComponent({ validationErrors: 'foo' })).toHaveClassName('nx-tooltip--validation-error');
+    expect(getShallowComponent()).toHaveClassName('nx-tooltip--validation-error');
+  });
+
+  it('sets the tooltips `open` prop to false if there are no validation errors', function() {
+    expect(getShallowComponent()).toHaveProp('open', false);
+    expect(getShallowComponent({ validationErrors: null })).toHaveProp('open', false);
+    expect(getShallowComponent({ validationErrors: [] })).toHaveProp('open', false);
```

Comment:
To mirror the test below, this should probably test the `title` as well as `open`

Reply:
> c45c435

> and title to null

That's not right though, and I'd hope the test isn't passing like that.  It sets it to empty string.  MUI `Tooltip` doesn't tolerate null or undefined

Reply:
> > [c45c435](https://github.com/sonatype/sonatype-react-shared-components/commit/c45c4354f7c8f54016e9ea928e33a4a7d250bef2)
> 
> > and title to null
> 
> That's not right though, and I'd hope the test isn't passing like that. It sets it to empty string. MUI `Tooltip` doesn't tolerate null or undefined

This is if there are no validation errors so the title of the NxTextInput is null. The NxTooltip takes that and converts it to empty string (to pass into MUI Tooltip) but this test is just on the NxTextInput